lesson-2-pods-deploymens-and-such/README.md

```diff
@@ -175,4 +175,13 @@ $ oc scale deployment httpbin-deployment --replicas 2
 
 ---
 
+## Follow-up
```

Comment:
Could we perhaps add this in a QuestionsAnswers.md file for the lecture? I think it would be nice to have for the first one too. This section might likely grow if we do more rounds of the workshop :slightly_smiling_face:

Reply:
that's a good idea, can do that!